lib/tests/2wp.js

```diff
@@ -105,13 +106,54 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic pegin v1 with the exact minimum value', async () => {
```

Comment:
Not sure if it's under your radar. But we should also have some tests that add the btc refund address to the op return payload.

Nothing will change since there won't be a refund, but we should check that the peg-in still works when we add that data.

And then some tests with invalid payloads, including invalid refund address data

Reply:
Yes.